src/common/storage/backends/s3/metadata.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "S3 Storage",
+  "configStructure": [
+    {
+      "name": "endpoint",
+      "displayName": "S3 Server URL",
+      "description": "The url to access the S3 Server. This should be a valid S3 endpoint. Examples- http://locahost:9000, https://s3.amazonaws.com",
```

Comment:
I would remove localhost as an example (trying not to deter its usage)

Reply:
I have replaced this with `https://play.min.io:9000`